lib/chef/provider/package.rb

```diff
@@ -88,14 +88,8 @@ def define_resource_requirements
           return
         end
 
-        # @todo: move the preseed code out of the base class (and complete the fix for Array of preseeds? ugh...)
-        if new_resource.response_file
-          if preseed_file = get_preseed_file(package_names_for_targets, versions_for_targets)
-            converge_by("preseed package #{package_names_for_targets}") do
-              preseed_package(preseed_file)
-            end
-          end
-        end
+        # for apt_package and dpkg_package
```

Comment:
Recommend removing this comment in favor of adding documentation to the `prepare_for_installation` method itself, what role does it perform for a package resource, and how it could be used generically.